src/math/Euler.js

```diff
@@ -280,10 +280,20 @@ THREE.Euler.prototype = {
 
 	},
 
-	toArray: function () {
+	toArray: function ( array, offset, withoutOrder ) {
 
-		return [ this._x, this._y, this._z, this._order ];
+		if ( array === undefined ) array = [];
+		if ( offset === undefined ) offset = 0;
 
+		array[ offset ] = this._x;
+		array[ offset + 1 ] = this._y;
+		array[ offset + 2 ] = this._z;
+
+		if( !withoutOrder ) {
```

Comment:
Here it is:
https://github.com/Wandalen/three.js/commit/80b7ac91566ee551c787c560e81a536d645b3c26